CONTRIBUTING.md

```diff
@@ -3,6 +3,29 @@
 First, please fork this repo and commit your changes to forked repo. From there make a Pull Request with your notebook submission keeping the following in mind:
 
 
+## Key Requirements
+
+First, you should check whether this notebook can be run using a Free Shared Tier instance. This is ideal for scenarios requiring few tables (<10), pipelines (<10) and less than 1GB of compressed storage. See the [limitations](https://docs.singlestore.com/cloud/getting-started-with-singlestoredb-cloud/shared-edition) to determine if your notebook can be run on a Free Shared Tier database. 
+
+If your database can be run using the Free Shared Tier (Starter Worksapce), you must:
+1. Mention this using a Markdown cell at the top of the notebook. See [example](https://www.singlestore.com/spaces/mongo-atlas-single-store-kai/). You should explicitly state that this notebook can run on both the Starter and Standard Workspaces. 
+2. Add "starter" to the tags in the meta.toml file below
+3. Use the following syntax when creating OR dropping your database. NOTE: When you create a Starter Workspace, a database is already being created for it. The only way you can drop that database linked to the Starter Workspace is by terminating the Workspace altogether. 
+
+```python
+shared_tier_check = %sql show variables like 'is_shared_tier'
+
+if not shared_tier_check or shared_tier_check[0][1] == 'OFF':
+    %sql CREATE DATABASE your_database_name
+    %sql DROP DATABASE your_database_name
```

Comment:
I think these lines are backwards, and the drop should also have an `IF EXISTS`. At the end of the notebook, you should add a cleanup cell that does the same shared tier check and drops the database.